language/Syntax.md

```diff
@@ -415,13 +415,13 @@ factorial =
     go 1
 ```
 
-The `where` keyword can also be used to introduce local declarations at the end of a value declaration:
+`where` clauses can be added at the end of a function declaration, and allow to bind variables that are visible throughout the function:
```

Comment:
Specifically, it's incorrect to say "function declarations", you can also do e.g.

```purescript
pi :: Number
pi = 4 * (sum (take 100 (taylorSeries arctan 1.0))
  where
  taylorSeries = [...]
  arctan = [...]
```

It would be incorrect to call this a "function declaration" because it isn't declaring a function; you can use a where clause in any kind of value declaration which is why "value declaration" is preferable. Also, "... allow to bind variables ..." is grammatically incorrect. Most importantly though, I think it would be best to make the structure of this sentence resemble that of the earlier sentence that tells you what the `let` keyword does as much as we reasonably can, so that it's as clear as possible that `let` and `where` do basically the same thing.